### DISCUSSION TITLE
essay/2024/02/23/vision-pro

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#178] [Comments]
# essay/2024/02/23/vision-pro

친구 집에서 비전 프로를 처음 경험해 봤습니다.

https://jeho.page/essay/2024/02/23/vision-pro.html

**Comment by ziw8** [comment]
부럽습니다!!
[첫번째 아이폰(2007)이 나올 시점에 출원된 애플의 특허](https://www.reddit.com/r/ipod/comments/1ajkc5i/2007_apple_vision_pro_patent/)가 무척이나 비전프로를 닮아있습니다.
(소름)

**Reply by BenjaminKim** [reply]
2007년부터 준비를 했었군요.
대단하다는 말밖엔... 😁

**Comment by lovesich** [comment]
아...... 얼간이..........................
